m1073496/overloook
# Feature/login

## What does this PR do?

- Creates login view
- Creates login SASS file for styling
- Creates fetch request for single customer
- Checks user input for valid username/password combo
- Alters index.js file where needed to account for changes in page load --> login

## How should it be tested in the UI?

- Reviewer can log in with various usernames and see the correct dashboard, search for and filter rooms, and add bookings
